docs/mllib-linear-methods.md

```diff
@@ -282,8 +277,8 @@ public class SVMClassifier {
     System.out.println("Area under ROC = " + auROC);
 
     // Save and load model
-    model.save(sc.sc(), "myModelPath");
-    SVMModel sameModel = SVMModel.load(sc.sc(), "myModelPath");
+    model.save(sc, "myModelPath");
```

Comment:
I think `sc.sc()` is required because `save` expects a `SparkContext` instead of `JavaSparkContext`. Though we should really overload it with `JavaSparkContext`.